test api/REST Client/API endpoints.http

```diff
@@ -0,0 +1,139 @@
+@clientid = 7e930f8e-fac4-43a6-a242-3a4079ac4553
+@clientsecret = clientSecret
+@scope = https://api.businesscentral.dynamics.com/.default
+@tenant = ffb102fa-38b1-4d6b-ad59-1192df615c1e
+@environmentName = Sandbox1
+@aPIPublisher = ciellos
+@aPIGroup = salesOrderApi
+@aPIVersion = v1.0
+@baseAPIUrl = {{baseUrl}}/v2.0/{{environmentName}}/api/{{aPIPublisher}}/{{aPIGroup}}/{{aPIVersion}}
+@entitySetName = items
+
```

Comment:
You can specify also "@pageSize" property for Api in this example.

Reply:
Code reviewed.